src/main/java/webserver/HttpProtocolSchema.java

```diff
@@ -0,0 +1,51 @@
+package webserver;
+
+import java.util.Objects;
+
+public class HttpProtocolSchema {
+    private final static String PROTOCOL_AND_VERSION_DELIMITER = "/";
+    private final static int PROTOCOL_AND_VERSION_SCHEMA_SIZE = 2;
+    private final static int PROTOCOL_SCHEMA_INDEX = 0;
+    private final static int VERSION_SCHEMA_INDEX = 1;
+    private final HttpProtocol protocol;
```

Comment:
```suggestion
    private final static int VERSION_SCHEMA_INDEX = 1;
    
    private final HttpProtocol protocol;
```
상수와 필드의 영역을 구분 지어주는 것이 어떨까요?